test/passing/tests/js_source.ml.ocp

```diff
@@ -98,18 +99,21 @@ let () =
   [%foo
     while () do
       ()
-    done [@foo]];
+    done
+    [@foo]];
   [%foo
     for x = () to () do
       ()
-    done [@foo]];
+    done
+               [@foo]];
```

Comment:
That's not right. The break should be avoided in this case while still breaking if the attribute would cross the margin.

Reply:
It looks like something we should fix in ocp-indent instead

Reply:
Agree. In the meantime, this PR would cause unwanted diffs so it shouldn't be merged.

This is a minor issue among a huge list so I think we can skip fixing it.